en/best-layout-practices.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Layout Best Practices - Design System
+_description: Best practices regarding layout creation and resizing configurations that result in the desired responsive behavior after code generation. 
+_keywords: Design Systems, Design Systems UX, UI kit, Ignite UI for Angular,  Design Kits for Angular, HTML, Design to HTML, UI kits
+---
+
+
+# Overview
+
+This topic provides design guidance regarding the positioning of Components in your app layouts and the setting of resizing rules that ensure good responsive app design.
+
+Each design tool (Figma, Sketch, XD) provides a unique way of configuring the positions and sizes of elements in the view. 
+
+In general these tools allow you to specify either a:
+
+## Absolute layout design
+
+  This is a design where all elements are ordered with absolute position on the view and have no additional positioning or sizing information. Their purpose is to show a design in a particular resolution only.
+
+  For such designs it is harder to determine a meaningful responsive layout as the designer's intent on how elements should react to resize of the view is not always clear.
+
+  The Design to Code algorithm will still attempt to build a responsive design based on the absolute layout positions as best as it can. For the specific rules and guidelines on the absolute to responsive layout refer to the [Absolute Layout Best Practices](./best-layout-practices.md#layout) section.
+  
+## Responsive layout design
+
+  This is a design that utilizes the specific tool's ability to mark layers as responsive to size changes and determine the responsive rules those layouts and their children should follow.
+    It is specific to each tool and you can find more information in their related topics:
+
+  - [Responsive layouts in Figma](./best-layout-practices-figma.md)
+  - [Responsive layouts in Sketch](./best-layout-practices-sketch.md)
+  - [Responsive layouts in XD](./best-layout-practices-xd.md)
+  
+  As with this approach there's more available information from the design on how things should react to resize it would always give generation results closer to the original design in the App builder. 
+  Due to this it's recommended to utilize the specific tool's responsive layouting option to get better results.
+
+# Absolute Layout - Best Practices
+
+This section provides design guidance regarding the positioning of Components in your app layouts. By sticking to the recommendations below, you will be able to generate an app effortlessly with the responsive behavior.
```

Comment:
Maybe remove `This section provides guidance`, there is the same phrase two sentences bellow